File: 프로그래머스/lv1/86491. 최소직사각형/최소직사각형.py
def solution(sizes):
    for i in range(len(sizes)):
        size = sizes[i].copy()
        sizes[i][0] = max(size)
        sizes[i][1] = min(size)
    
    answer = max([i[0] for i in sizes]) * max([i[1] for i in sizes])

    return answer

File: 프로그래머스/1/86491. 최소직사각형/최소직사각형.js
function solution(sizes) {
    var answer = [0, 0];
    sizes.forEach(size => {
        const min = Math.min(...size)
        const max = Math.max(...size)
        if (answer[0] < max) {
            answer[0] = max
        }
        if (answer[1] < min) {
            answer[1] = min
        }
    })
    return answer.reduce((acc, cur) => acc *= cur, 1);
}